### PR TITLE
mkvtoolnix: 51.0.0 -> 53.0.0

### PR DESCRIPTION
**Pull request by emmanuelrosa**
###### Motivation for this change

This is a version bump for mkvtoolnix (and consequently libebml).

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Determined the impact on package closure size (by running `nix path-info -S` before and after). Before: 1184201016. After 1186309064
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rnhmjoj** [comment]
@ryantm any idea why would @r-ryantm stop updating this package? It used to work until recently.

File: pkgs/development/libraries/libebml/default.nix [hunk 1]
```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "libebml";
-  version = "1.4.0";
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/libebml/default.nix:5]: Is this unrelated? If not put it into a separate commit.

**Comment by ryantm** [comment]
> @ryantm any idea why would @r-ryantm stop updating this package? It used to work until recently.

Failed to build. Error msg: 
```
** The internal version of the libEBML library is supposed to be used,
*** but it was not found in "lib/libebml. If this is a clone from the
*** git repository then submodules have to be initialized with the
*** following two commands:
***
*** git submodule init
*** git submodule update
```

More details at https://r.ryantm/log/

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 112384` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages marked as broken and skipped:
</summary>
<ul>
<li>libsForQt512.elisa</li>
<li>libsForQt514.elisa</li>
</ul>
</details>
<details>

  <summary>1 package failed to build and are new build failure:</summary>
  <ul>
<li>obs-ndi: log was empty</li>
  </ul>
</details>
<details>
<summary>
31 packages built:
</summary>
<ul>
<li>gerbera</li>
<li>libebml</li>
<li>liblinphone</li>
<li>libmatroska</li>
<li>libsForQt5.elisa (libsForQt515.elisa ,plasma5Packages.elisa)</li>
<li>libsForQt5.phonon-backend-vlc (libsForQt515.phonon-backend-vlc ,plasma5Packages.phonon-backend-vlc)</li>
<li>libsForQt512.phonon-backend-vlc</li>
<li>libsForQt514.phonon-backend-vlc</li>
<li>libvlc</li>
<li>linphone</li>
<li>mediastreamer</li>
<li>mediastreamer-openh264</li>
<li>megaglest</li>
<li>minitube</li>
<li>mkvtoolnix</li>
<li>mkvtoolnix-cli</li>
<li>mpvScripts.convert</li>
<li>msilbc</li>
<li>obs-gstreamer</li>
<li>obs-move-transition</li>
<li>obs-studio</li>
<li>obs-v4l2sink</li>
<li>obs-wlrobs</li>
<li>pympress</li>
<li>python37Packages.python-vlc</li>
<li>python38Packages.python-vlc</li>
<li>python39Packages.python-vlc</li>
<li>reaper</li>
<li>strawberry</li>
<li>tribler</li>
<li>vlc (vlc_qt5)</li>
</ul>
</details>

**Comment by emmanuelrosa** [comment]
> > @ryantm any idea why would @r-ryantm stop updating this package? It used to work until recently.
> 
> Failed to build. Error msg:
> 
> ```
> ** The internal version of the libEBML library is supposed to be used,
> *** but it was not found in "lib/libebml. If this is a clone from the
> *** git repository then submodules have to be initialized with the
> *** following two commands:
> ***
> *** git submodule init
> *** git submodule update
> ```
> 
> More details at https://r.ryantm/log/

That's caused by mkvtoolnix now requiring libebml 1.4.1. Since Nixpkgs contains libebml 1.4.0, the mkvtoolnix configure script fails to find the library and attempts to fall back to the bundled copy, which it also fails to find.

This PR also updates libebml to address this problem.
